### PR TITLE
Cache

### PR DESCRIPTION
**Pull request by Leits**
Many thanks to @mykhaly. :)

### CODE REVIEW

**Comment by Leits** [comment]
PL для відображення і відстеженя. Не готовий до прийняття.

File: op_robot_tests/tests_files/meatTender.robot [hunk 1]
```diff
@@ -41,9 +47,27 @@ ${broker}       Quinta
...
+  \  Звірити дату тендера  ${username}  ${USERS.users['${tender_owner}'].initial_data}  tenderPeriod.startDate
+
+Відображення закінчення періоду прийому пропозицій оголошеного тендера
+  [Tags]   ${USERS.users['${viewer}'].broker}: Відображення основних даних оголошеного тендера
+  ...      viewer
+  ...      ${USERS.users['${viewer}'].broker}
+  ...      minimal
+  ${usernames}=  Create List  ${viewer}  ${provider}  ${provider1}
+  :FOR  ${username}  IN  @{usernames}
+  \  Звірити дату тендера  ${username}  ${USERS.users['${tender_owner}'].initial_data}  tenderPeriod.endDate
```
Comment (selurvedu) [inline op_robot_tests/tests_files/meatTender.robot]: Should be `auctionPeriod.startDate`, dont forget to change the title.

File: op_robot_tests/tests_files/brokers/openprocurement_client.robot [hunk 2]
```diff
@@ -38,305 +36,210 @@ Library  openprocurement_client_helper.py
...
+  ${bid_id}=  Get Variable Value   ${USERS.users['${username}'].bidresponses['resp'].data.id}
+  ${tender}=  Пошук тендера по ідентифікатору  ${username}  ${tender_uid}
+  ${tender}=  set_access_key  ${tender}  ${USERS.users['${username}'].bidresponses['resp'].access.token}
+  ${response}=  Call Method  ${USERS.users['${username}'].client}  upload_bid_document  ${path}  ${tender}  ${bid_id}
+  ${uploaded_file} =  Create Dictionary   filepath  ${path}   upload_response  ${response}
   Log  ${response}
   Log object data   ${uploaded_file}
   [return]  ${uploaded_file}
+  # TODO:
+  # [Arguments]  ${username}  ${path}  ${tender_uid}
```
Comment (selurvedu) [inline op_robot_tests/tests_files/brokers/openprocurement_client.robot]: Already done.

**Comment by selurvedu** [comment]
Reviewed 1 of 1 files at r1, 2 of 2 files at r2, 1 of 1 files at r3, 3 of 3 files at r4, 1 of 1 files at r5, 1 of 1 files at r6, 1 of 1 files at r7, 1 of 1 files at r8.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/50)*

**Pull request merged**